docs2/site/docs/getting-started/schema-types.md

```diff
@@ -94,9 +98,56 @@ enum Episode {
 }
 ```
 
+**.NET**
+
+Consider the equivalent `enum` in .NET:
+
+```csharp
+public enum Episodes
+{
+    NEWHOPE = 4,
+    EMPIRE = 5,
+    JEDI = 6
+}
+```
+
+Compare the two implementations. GraphQL does not specify backing values for members of its enums. The name of each member _is_ the value.
+
 **GraphQL .NET**
 
-You can manually create the `EnumerationGraphType`.
+GraphQL.NET provides two methods of defining GraphQL enums.
+
+You can use `EnumerationGraphType<TEnum>` to automatically generate values by providing a .NET `enum` for `TEnum`. The `Name` will default to the .NET Type name, which you can override in the constructor. The `Description` will default to any `System.ComponentModel.DescriptionAttribute` applied to the enum type. The `DeprecationReason` will default to any `System.ObsoleteAttribute` applied to the enum type. By default, the name of each enum member will be converted to CONSTANT_CASE. Override `ChangeEnumCase` to change this behavior. Apply a `DescriptionAttribute` to an enum member to set the GraphQL `Description`. Apply an `ObsoleteAttribute` to an enum member to set the GraphQL deprecation reason.
+
+```csharp
+[Description("The Star Wars movies.")]
+[Obsolete("Optional. Sets the GraphQL DeprecationReason for the whole enum.")]
+public enum Episodes
+{
+    [Description("Episode 1: The Phantom Menace")]
+    [Obsolete("Optional. Sets the GraphQL DeprecationReason for this member.")]
+    PHANTOMMENACE = 1,
+
+    [Description("Episode 4: A New Hope")]
+    NEWHOPE  = 4,
+
+    [Description("Episode 5: The Empire Strikes Back")]
+    EMPIRE  = 5,
+
+    [Description("Episode 6: Return of the Jedi")]
+    JEDI  = 6
+}
+
+public class EpisodeEnum : EnumerationGraphType<Episodes>
+{
+}
+```
+
+You can also manually create the `EnumerationGraphType`. Advantages of this method:
+
+- The GraphQL enum need not map to a specific .NET `enum`. You could, for instance, build the enum from one of the alternate methods of defining discrete sets of values in .NET, such as classes of constants or static properties.
+- You can manually add description and deprecation reasons.  This may be useful if you do not control the source code for the enum.
```

Comment:
```suggestion
- You can manually add descriptions and deprecation reasons. This may be useful if you do not control the source code for the enum.
```